### ISSUE TITLE
TTL: when enabling the TTL functionality the index only gets added to the default DB

### ISSUE THREAD

**Issue by gwvandesteeg** [neo4j-contrib/neo4j-apoc-procedures#1588]
## Expected Behavior (Mandatory)

Expected to have the TTL index created on those databases that didn't already have the index.

## Actual Behavior (Mandatory)

Any additional databases/graphs created using CREATE DATABASE don't have the TTL index added even when the image is restarted.

## How to Reproduce the Problem

1. Start a docker instance with apoc.ttl.enabled set to true
1. Create an additional database
1. Restart the docker instance

### Simple Dataset (where it's possibile)

```
:use neo4j
CALL db.indexes();
:use system
CREATE DATABASE missingttl;
:use missingttl
CALL db.indexes();
```
Reboot the instance and reconnect
```
:use missingttl
CALL db.indexes();
```

### Steps (Mandatory)

  1. See above

## Screenshots (where it's possibile)

nil

## Specifications (Mandatory)

nil

Currently used versions

### Versions

  - OS: Ubuntu 19.04
  - Docker: 19.03
  - Neo4j: 4.1.0
  - Neo4j-Apoc: 4.1.0.0

**Comment by lqst** [comment]
What happens if you create a new node in the 'missingttl' database and set it to expire? For instance:
`create (p:Person{id:1}) return p`
`match (p:Person{id:1}) with p call apoc.ttl.expireIn(p, 1, 's') return p`

At least when I tested, the TTL index is then added.

**Comment by lqst** [comment]
Scrap that, you only need to wait for the schedule to kick in before the index is created (runs every 60s by default and can be configured with apoc.ttl.schedule).

**Comment by gwvandesteeg** [comment]
Yup, can confirm that the index is created after a TTL schedule run.  Nothing to do here then code wise, just need to update the documentation that this is the behaviour.

**Comment by lqst** [comment]
Thank you for confirming, I will have a stab later at updating the docs.

**Comment by vga91** [comment]
Fixed by #1611
